deploy/certificate_config.yaml

```diff
@@ -0,0 +1,156 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: service-apis-admission
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
+  namespace: service-apis
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: service-apis-admission
+  labels:
+    name: service-apis
+    version: 0.0.1
+rules:
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - validatingwebhookconfigurations
+    verbs:
+      - get
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: service-apis-admission
+  annotations:
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
+  namespace: service-apis
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: service-apis-admission
+subjects:
+  - kind: ServiceAccount
+    name: service-apis-admission
+    namespace: service-apis
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: service-apis-admission
+  annotations:
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
+  namespace: service-apis
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - secrets
+    verbs:
+      - get
+      - create
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: service-apis-admission
+  annotations:
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
+  namespace: service-apis
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: service-apis-admission
+subjects:
+  - kind: ServiceAccount
+    name: service-apis-admission
+    namespace: service-apis
+---
+# TODO(cmluciano): Use helm-based configuration.
```

Comment:
Agree with this TODO but maybe we should remove it from this file and create a follow up issue instead. Alternatively Kustomize might work here - we've got some uses of it elsewhere.